### PR TITLE
[patch] Fix the condition for db2 setup for Manage app

### PR DESCRIPTION
**Pull request by sbudhira**
Fixed the condition for db2 setup for Manage app.

Tested scenarios 
1)  DB2 install/setup & Manage in cluster
2)  BYO JDBC cfg (external DB) with Manage

### CODE REVIEW

File: tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2 [hunk 1]
```diff
@@ -11,12 +11,12 @@
     - input: "$(params.mas_app_channel_manage)"
       operator: notin
       values: [""]
-    - input: "$(params.db2_action_system)"
+    - input: "$(params.db2_action_manage)"
       operator: in
       values: ["install"]
     - input: "$(params.db2_action_manage)"
```
Comment (durera) [inline tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2]: 3rd condition here is redundant ... condition 2 will only be true if db2_action_manage == "install" ... so there's no need to also add a condition where db2_action_manage is also not "byo"

```
  when:
    - input: "$(params.mas_app_channel_manage)"
      operator: notin
      values: [""]
    - input: "$(params.db2_action_manage)"
      operator: in
      values: ["install"]
```

achieves the same thing doesn't it, or am I missing something?

**Review by durera — DISMISSED** [review]

File: tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2 (hunk 1 above)
Comment (whitfiea) [reply tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2]: I thought it was saying that we want to do the setup for manage only if a) manage is being installed and b) it is not a BYO DB2. If it is a manage install using a BYO DB2 then you don't want to the run the setup??

Comment (sbudhira) [reply tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2:17]: The first two conditions were original conditions. I added third one.

Comment (whitfiea) [reply tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2]: So further clarification is that "db2_action_manage" "install" means install db2 for manage. So if that is true then BYO would always be false, so no need for the check. @sbudhira I don't think we need that third condition.

Comment (sbudhira) [reply tekton/src/pipelines/taskdefs/apps/db2-setup-manage.yml.j2]: @whitfiea If DB2 is installed in the cluster then set up DB2. If using BYO jdbc (external DB db2, oracle or sql server ) , then don't setup DB2.

**Review by terenceq — APPROVED** [review]

**Pull request merged**